integration/buttons.spec.ts

```diff
@@ -9,15 +9,9 @@ const Title = Selector("h1");
 const Button = Selector("button");
 const ButtonBack = Selector("[data-testid='back-button']");
 const ButtonBackRed = Selector("[data-testid='red-back-button']");
-const CloseTab = ClientFunction(() => window.close());
 const ProgressBar = Selector("[data-testid='progress-bar']");
 
 test("should lead to pages correctly", async (t) => {
-  // Check documentation button open in tab
-  await t.click(Selector("[data-testid='config-file-docs-button']"));
-  await t.expect(Title.textContent).contains("Config File");
-  await CloseTab();
```

Comment:
maybe shouldn't test on live url response, shifting this to unit test and test only for attributes match